bigbluebutton-web/src/java/org/bigbluebutton/api/messaging/MessageToJson.java

```diff
@@ -35,6 +35,7 @@ public static String createMeetingMessageToJson(CreateMeetingMessage msg) {
 		payload.put(Constants.DURATION, msg.duration);
 		payload.put(Constants.AUTO_START_RECORDING, msg.autoStartRecording);
 		payload.put(Constants.ALLOW_START_STOP_RECORDING, msg.allowStartStopRecording);
+		payload.put(Constants.WEBCAMS_ONLY_FOR_MODERTORS, msg.webcamsOnlyForModerator);

```

Comment:
All of the other instances leave off the "s" at the end. Should probably remove to match.